robot/impl/local_robot.go

```diff
@@ -1147,12 +1147,14 @@ func (r *localRobot) Reconfigure(ctx context.Context, newConfig *config.Config)
 	}
 }
 
-func walkConvertedAttributes[T any](pacMan packages.ManagerSyncer, convertedAttributes T, allErrs error) (T, error) {
+func walkConvertedAttributes[T any](
+	pacMan packages.ManagerSyncer, packageMap map[string]string, convertedAttributes T, allErrs error,
```

Comment:
we need to pass this packageMap now into the walker because it is used by the path visitor to find and replace filepath placeholders based on real packages in config.